core-chart/templates/controlplanes/its.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   backend: shared
   type: {{ $cp.type | default "vcluster" }}
-  postCreateHook: its
+  postCreateHook: {{ (eq $cp.install_clusteradm false) | ternary "its-without-clusteradm" "its-with-clusteradm" }}

```

Comment:
What happens here if the user has not provided a value for `install_clusteradm` in the `cp` ?

Reply:
absence of the value will evaluate as NOT == false and thus install OCM

Reply:
Thank you.